atlasdb-container-test-utils/src/test/java/com/palantir/atlasdb/containers/CassandraEnvironmentTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.containers;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import java.util.Map;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
+
+public class CassandraEnvironmentTest {
+    @Rule
+    public final EnvironmentVariables environment = new EnvironmentVariables();
+
+    @Test
+    public void testDefaults() {
+        environment.set(CassandraEnvironment.CASSANDRA_VERSION, null);
```

Comment:
Yes, otherwise if non-default environment variables happen to exist, this test fails. (The test uses a "fake" environment, but pre-populates it with the current environment)